app/Food.php

```diff
@@ -18,4 +18,14 @@ public function orderItems()
     {
         return $this->morphMany(OrderItem::class, 'itemtable');
     }
+    
+    /**
+     * Get the category for the food.
+     *
+     * @return relationship
```

Comment:
I'm sorry, I don't have a clear feedback. You should return type relationship. example:
@return \Illuminate\Database\Eloquent\Relations\HasMany.